src/authentication/oauth2Auth.ts

```diff
@@ -750,6 +753,6 @@ export class Oauth2Auth extends AlfrescoApiClient {
 
         this.storage.removeItem('nonce');
         this.storage.removeItem('jwks');
-        this.storage.removeItem('discovery');
+        // this.storage.removeItem('discovery');
```

Comment:
Can you please r move the commented code?

Reply:
we removed the ‘discovery’ flag from the local storage in the PR
https://github.com/Alfresco/alfresco-js-api/blob/development/src/authentication/oauth2Auth.ts#L753
because as you can see in
https://github.com/Alfresco/alfresco-ng2-components/blob/development/lib/core/login/components/login.component.ts#L221
https://github.com/Alfresco/alfresco-ng2-components/blob/development/lib/core/services/authentication.service.ts#L324
It raises an error only we invalidated the sessions which happens with every logout. So my question is why is it there in the first place?